bin/main.rs

```diff
@@ -305,13 +327,13 @@ fn configure_tracing(
         .with(log_layer);
     if !tracing_enabled && tracing_endpoint.is_none() {
         if let Err(e) = tracing::subscriber::set_global_default(subscriber) {
-            eprintln!("Logger/tracer was already initted, continuing: {}", e);
+            eprintln!("Logger/tracer was already initted, continuing: {e}");
```

Comment:
grammar nit (super tiny): should probably use `initialized` here instead of `initted`. Less likely to cause confusion for non-native speakers.

Reply:
I think this was a clippy auto-fix, but will update the text here!

- [ ] "initted" -> "initialized"